ntbs-service/Pages/Notifications/Edit/PreviousHistory.cshtml

```diff
@@ -5,49 +5,49 @@
-                            </span>
-                            <input v-on:blur="validate" class="nhsuk-input nhsuk-input--width-10" asp-for="PatientTBHistory.PreviousTBDiagnosisYear" type="text">
-                        </nhs-form-group>
-                    </div>
+    <div class="nhsuk-form-group">
+    <nhs-fieldset aria-describedby="tb-diagnosis-year-error">
+        <nhs-fieldset-legend nhs-legend-size="Standard"> The patient has had TB in the past </nhs-fieldset-legend>
+        <div class="nhsuk-radios govuk-radios--conditional" data-module="govuk-radios">
+            <div class="nhsuk-radios__item">
+                <input asp-for="PatientTBHistory.NotPreviouslyHadTB" class="nhsuk-radios__input" type="radio" value="false" data-aria-controls="conditional-previously-had-tb-conditional">
+                <label class="nhsuk-label nhsuk-radios__label" asp-for="PatientTBHistory.NotPreviouslyHadTB">
+                    Yes
+                </label>
+            </div>
+            
+            <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-previously-had-tb-conditional">
+                <validate-input model="PreviousHistory" property="PreviousTBDiagnosisYear" shouldvalidatefull="@fullValidation" inline-template>
+                    @{
+                        var previousTBDiagnosisError = !Model.IsValid("PatientTBHistory.PreviousTBDiagnosisYear");
+                        var previousTBDiagnosisGroupState = previousTBDiagnosisError ? Error : Standard;
+                    }
+                    <nhs-form-group nhs-form-group-type="@previousTBDiagnosisGroupState" aria-describedby="previous-tb-diagnosis-year-error">
+                        <label nhs-label-type="Standard" asp-for="PatientTBHistory.PreviousTBDiagnosisYear"> Year of diagnosis </label>
+                        <span nhs-span-type="ErrorMessage" id="previous-tb-diagnosis-year-error" ref="errorField"
+                            asp-validation-for="PatientTBHistory.PreviousTBDiagnosisYear"></span>
+                        <input ref="inputField" v-on:blur="validate" is-error-input=@previousTBDiagnosisError 
+                            nhs-input-type="Standard" classes="nhsuk-input--width-4" asp-for="PatientTBHistory.PreviousTBDiagnosisYear">
+                    </nhs-form-group>
                 </validate-input>
-                <div class="nhsuk-radios__item">
-                    <input asp-for="PatientTBHistory.NotPreviouslyHadTB" class="nhsuk-radios__input" type="radio" value="true">
-                    <label class="nhsuk-label nhsuk-radios__label" asp-for="PatientTBHistory.NotPreviouslyHadTB">
-                        No
-                    </label>
-                </div>
             </div>
-        </nhs-fieldset>
-    </div>
-    <br/>
-    <partial name="_PaginationPartial">
-</date-comparison>
+
+            <div class="nhsuk-radios__item">

```

Comment:
We can use the tag helpers here i think?

Reply:
tag helpers don't play nice with govuk-*-conditional :(